addons/website_event/views/event_templates_page_registration.xml

```diff
@@ -351,7 +432,7 @@
             <div class="modal-content">
             <div class="modal-header">
                 <div class="o_wevent_registration_title modal-title fs-5" id="staticBackdropLabel">Tickets</div>
-                <div t-if="len(event.event_ticket_ids) &gt; 1" class="o_wevent_price_range ms-2"/>
+                <div t-if="len(event.event_ticket_ids) &gt; 2" class="o_wevent_price_range ms-2"/>
```

Comment:
Seems weird actually. We want to show it if there's 3 or more tickets? Not 2 or more?